server/src/routes/controllers/user.ts

```diff
@@ -0,0 +1,23 @@
+// SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import { Request, Response } from 'express';
+import { TeacherCourses, getTeacherCourses } from '../../services/user';
+
```

Comment:
I think it is common practice to have the controllers located at src/controllers instead of src/routes/controllers

Reply:
In that case `src/routes/index.ts` should probably also be moved to `src/routes.ts` and the `src/routes` directory can be deleted completely. I think that would be better too but we'll have to discuss that first.